axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankreconciliation/BankReconciliationService.java

```diff
@@ -210,13 +210,17 @@ public void generateMovesAutoAccounting(BankReconciliation bankReconciliation)
 
         for (BankStatementRule bankStatementRule : bankStatementRules) {
 
-          if (Boolean.TRUE.equals(
-              new GroovyScriptHelper(scriptContext)
-                  .eval(
-                      bankStatementRule
-                          .getBankStatementQuery()
-                          .getQuery()
-                          .replaceAll("%s", "\"" + bankStatementRule.getSearchLabel() + "\"")))) {
+          if (bankStatementRule != null
+              && bankStatementRule.getBankStatementQuery() != null
+              && bankStatementRule.getBankStatementQuery().getQuery() != null
```

Comment:
Could you use com.google.common.base.Strings.isNullOrEmpty instead of != null, it's better when we have a string.